practipy/gcloud.py

```diff
@@ -112,7 +112,7 @@ def download_files(
     Note: paths are relative to `gs://<bucket_name>`!.
     """
 
-    bucket = gcs.Client(project=project).get_bucket(bucket)
+    bucket = gcs.Client(project=project).bucket(bucket_name)
```

Comment:
The `bucket_name` is an unrelated change because my IDE complained about assigning a bucket to a `str`.